app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -9,15 +9,18 @@ body {
 }
 
 // TODO: Hadi- Move this to specific file to only be accessible where needed
-#rooms-card {
+.rooms-card {
   &:hover {
       background: rgba(26, 69, 74, 0.2)
   }
 }
 
-#room-card-top {
+.room-card-top {
   &:hover {
     cursor: pointer;
-}
+  }
 }
 
+.tab-content{
+  padding: 0%;
```

Comment:
What's this for?

Reply:
This is for a div that was created by the <Tabs> in RoomsTabs.jsx. 
Tabs creates this div and then adds some padding which makes the room cards not aligned with the tabs (it creates the gaps we talked about above). In order to remove the gap like we discussed, I found that this was the only way